specification/dartLangSpec.tex

```diff
@@ -18209,16 +18209,19 @@ \section{Libraries and Scripts}
 \LMHash{}%
 A library may optionally begin with a \Index{script tag}.
 Script tags are intended for use with scripts (\ref{scripts}).
-A script tag can be used to identify the interpreter of the script to whatever computing environment the script is embedded in.
+A script tag can be used to identify the interpreter of the script to
+whatever computing environment the script is embedded in.
 The script tag must appear before any whitespace or comments.
 A script tag begins with \lit{\#!} and ends at the end of the line.
 Any characters that follow \lit{\#!} in the script tag are ignored by
 the Dart implementation.
 
 \LMHash{}%
 Libraries are units of privacy.
-A private declaration declared within a library $L$ can only be accessed by code within $L$.
-Any attempt to access a private member declaration from outside $L$ will cause a method, getter or setter lookup failure.
+A private declaration declared within a library $L$
+can only be accessed by code within $L$.
+Any attempt to access a private member declaration from outside $L$
+will cause a method, getter or setter lookup failure.
```

Comment:
This sentence is confusing. If I do `c._x`, it's a matter of intent whether I'm "attempt"ing to look up an inaccessible private member of `C` (the type of `C`) ... or the (potentially accessible) private member of `C`'s superclass. Both of them will find the superclass member if it's accessible, so not necessarily a "lookup failure".
We can't determine intent from syntax.

Probably just remove it. Maybe:

> A name starting with an underscore cannot refer to a declaration from another library.

Reply:
Right, it's better to simply remove that sentence. Done.